var/spack/repos/builtin/packages/gromacs/package.py

```diff
@@ -82,4 +82,8 @@ def cmake_args(self):
         else:
             options.append('-DCMAKE_BUILD_TYPE:STRING=Release')
 
+        if '+cuda' in self.spec:
+            options.append('-DGMX_GPU=ON')
+            options.append('-DCUDA_TOOLKIT_ROOT_DIR='+self.spec['cuda'].prefix)
```

Comment:
Could be a difference in the version of flake8 you're using. I've seen this as well.